lib/smart_proxy_remote_execution_ssh/api.rb

```diff
@@ -4,6 +4,98 @@ class Api < ::Sinatra::Base
       get "/pubkey" do
         File.read(Ssh.public_key_file)
       end
+      post "/session" do
+        if env["HTTP_CONNECTION"] != "upgrade" or env["HTTP_UPGRADE"] != "raw" then
```

Comment:
The value of the "upgrade" header is a list, and we need to check whether "raw" is in it.

Reply:
`then` is not really needed when the `if` spans multiple lines

Reply:
Removed the `then`s, thanks.  (The first comment about the upgrade header is still open.)